src/NServiceBus.Transport.SQS.Tests/MessageExtractionTests.cs

```diff
@@ -104,6 +142,7 @@ static IEnumerable<TestCaseData> GenerateTestCases(bool passMessageIdInMessageAt
             var senderTransportMessage = new TransportMessageBuilder()
                 .If(passMessageIdInNsbHeaders, t => t
                     .WithHeader(Headers.MessageId, nsbMessageIdPassedThroughHeaders))
+                .WithHeader(Headers.EnclosedMessageTypes, "Enclosed message type")
```

Comment:
NOTE: Message-Id is [one of the headers that is used to decide whether this is a valid non-native message](https://github.com/Particular/NServiceBus.AmazonSQS/pull/2341/files#diff-a6c1b75b370fb59b462c93cd7112b57483394ae2de9d14d11a2e62b23749a4ddR448). If it's not included then we need the other header, which is enclosed message type